components/cover/template.rst

```diff
@@ -92,20 +92,18 @@ with the ``cover.template.publish`` action.
     on_...:
       - cover.template.publish:
           id: template_cov
-          state: OPEN
+          position: OPEN
 
       # Templated
       - cover.template.publish:
           id: template_cov
-          state: !lambda 'return COVER_OPEN;'
+          position: !lambda 'return COVER_OPEN;'
```

Comment:
This still applies to the action. This was not removed/changed.

Reply:
You are correct, this compiles fine. I will revert this change.